packages/x-archetype-utils/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@empathyco/x-archetype-utils",
+  "version": "0.0.1",
+  "description": "Utilities for x-archetype",
+  "author": "Empathy Systems Corporation S.L.",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/empathyco/x/tree/main/packages/x-archetype-utils#readme",
+  "keywords": [
+    "utils",
+    "x",
+    "x-archetype"
+  ],
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/empathyco/x.git",
+    "directory": "packages/x-archetype-utils"
+  },
+  "engines": {
+    "node": ">=14.14.0"
+  },
+  "scripts": {
+    "prebuild": "rimraf dist ./*.tgz",
+    "build": "concurrently \"npm run build:*\"",
+    "build:cjs": "node_modules/.bin/tsc --project tsconfig.cjs.json",
```

Remove node_modules part in all builds

```suggestion
    "build:cjs": "tsc --project tsconfig.cjs.json",
```